src/Tools/dotnet-trace/Extensions.cs

```diff
@@ -12,7 +12,41 @@ namespace Microsoft.Diagnostics.Tools.Trace
 {
     internal static class Extensions
     {
+        public static string CLREventProviderName = "Microsoft-Windows-DotNETRuntime";
+
         private static EventLevel defaultEventLevel = EventLevel.Verbose;
+        // Keep this in sync with runtime repo's clretwall.man
+        private static Dictionary<string, long> CLREventKeywords = new Dictionary<string, long>(StringComparer.InvariantCultureIgnoreCase)
+        {
+            { "gc", 0x1 },
```

Comment:
nit: Could have PascalCase the keywords here :(

Reply:
i thought about it but since these keys won't be displayed on the console.... meh :D

Reply:
I would have expected them to show up on the `--help` instead of expecting users to go to the documentation in the repo.